### DISCUSSION TITLE
【教程】绳网常见问题＆解决方法

### DISCUSSION THREAD

**Discussion by March-777th** [share121/inter-knot#4990] [教程]
最近新人很多啊，我们绳网也是蒸蒸日上了<img src="https://bbs-static.miyoushe.com/static/2024/07/31/2dafa973b0fb29e5342403c4ead8af29_7443113916298582946.png" alt="三月七 比划" width="75" >
虽然绳网有[官方文档](https://d.inot.top/)，但存在感。。。而且一些小白问题也没有解答，所以特开一贴回答下新人容易碰到的问题
## 如何发送表情包？
- 安卓端使用**via**或者其它能安装脚本的浏览器[点这里](https://dave-12138.cn/static/Tampermonkey/miyoushe-emot-to-github.user.js)
- 电脑端需下载[油猴](https://www.tampermonkey.net/)或者其它能运行脚本的拓展，启用拓展后[点这里](https://dave-12138.cn/static/Tampermonkey/miyoushe-emot-to-github.user.js)进行安装，如果出现跨域提示点击**总是允许**就好
## 怎么改头像？
- 首先进入github官网，点击右上角头像
![IMG_20240916_205726](https://github.com/user-attachments/assets/3c1e1fb6-ea0e-46c8-9c08-d6d719ff03ad)
- 点击setting进入用户设置
- 之后下拉找到自己头像，点击左上角Edit即可更换
![d965d1188c9761b86809cc72e6415f0](https://github.com/user-attachments/assets/2be27906-f41a-4e43-b7ce-78590a57b3d2)

## 访问速度太慢怎么办？

- 以前发过一篇#1138，这边放个更快~~更贵~~的梯子[EFCloud](https://www.efcloud.dev/#/stage/dashboard)

## 怎么看不到回复？
- ~~绳网是这样的，需要退出重进一下才能看到回复和新帖~~
- 刷新下就好了
## 图片上传失败？
- 上传图片后需要等待**Uploading**
![IMG_20240916_204233](https://github.com/user-attachments/assets/abc3694f-5930-4770-b482-589e38614e56)
- 发送的图片不可大于**10mb**
## 邮箱被别人的回复塞满？
- 回复之后把**订阅/Subscribe**关一下
## 怎么删帖子？
- 答：删不了，请谨慎发言
## 如何提升绳网等级？
- 绳网的用户等级是用户的github公共仓库数，所以请努力吧~~其实多fork几个就行~~
## 如何使你的贴子更好看？
- [Markdown语法](https://d.inot.top/guides/markdown-syntax/)，支持原作者[纳西妲](https://naxida.me/)谢谢喵<img src="https://img-static.mihoyo.com/communityweb/upload/8aee1ff2c6508b43813237163cacec1c.png" alt="三月七_加油" width="75" >
##### ~~快跑，里面机器人比活人多~~
- 启动机场，加入[绳网频道](https://disboard.org/zh-cn/server/1273078781241987134)，证明你不是机器人，然后就进去辣
- 欢迎加入绳网dc频道喵<img src="https://img-static.mihoyo.com/communityweb/upload/8aee1ff2c6508b43813237163cacec1c.png" alt="三月七_加油" width="75" >

**Comment by KELERSAMA** [comment]
小三月你是一个一个一个天使

**Reply by March-777th** [reply]
哼↗哼↘哼~<img src="https://img-static.mihoyo.com/communityweb/upload/66f5c4854e9212e50fae7aeb97995a73.png" alt="三月七_开心" width="75" >

**Comment by passbyDFX** [comment]
哇哦

**Comment by LV0Incapableperson** [comment]
如果移动端用户找不到更改头像的edit 可以试试用浏览器的"电脑模式"按照小三月的方法去改头像 
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by yihangyingzi** [comment]
好耶<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >

**Comment by LeoneMafia** [comment]
学废了学废了，谢谢楼主

**Comment by d2531987182** [comment]
多谢多谢<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Comment by nostalgiatan** [comment]
咦，看到一个有意思的工具箱

**Reply by nostalgiatan** [reply]
下次有什么有意思的东西可以直接@ 我，
我来视察👀，我来选择👌，我来闲逛😎

**Reply by KELERSAMA** [reply]
ewq 大佬

**Reply by March-777th** [reply]
# 佬<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Comment by zzz20100624** [comment]
牛牛牛

**Comment by Labyrinth0419** [comment]
三月老师可爱捏

**Comment by Vilei2000** [comment]
前排售卖燃油饮、枫达、汽水、口嚼酒、团子牛奶、青衣喝过的茶。
🍾🍹🧊🧊💦🥛🍺🧃🍵🛒🛒🚚大钢牙

**Comment by Yizakl** [comment]
# 大佬
窝的iOSapp教程都没人看

**Reply by nostalgiatan** [reply]
啊嘞啊嘞，可能是我没刷到吧，抱歉。

**Comment by EcoFur** [comment]
> # 大佬
> 
> 窝的iOSapp教程都没人看

# __**大佬！**__

**Comment by 1talan** [comment]
老师，如何查看自己的历史帖子呢？

**Comment by EcoFur** [comment]
如果你平时不用GitHub，可以到自己的GitHub主页看到对这个仓库发的贡献


Sent from my Apple Watch

**Comment by 123xiaojun** [comment]
大佬！

**Comment by mwqf** [comment]
非常感谢，也是弄好了。

**Comment by kdhfkc** [comment]
水

**Comment by 323157364** [comment]
谢谢大佬

**Comment by coolwide** [comment]
![Uploading 1723608875426.png…]()

**Comment by fengkong-cjb** [comment]
三月七的礼物好康😁

**Comment by boziqwq** [comment]
好！<img src="https://img-static.mihoyo.com/communityweb/upload/6b4c9cb96726540a6ab37d380e008a6e.png" alt="抱" width="75" >

**Comment by 772099448** [comment]
太感谢了

**Comment by xiwu2** [comment]
谢谢你小三月😘

**Reply by xiwu2** [reply]
表情包挺好玩的🌚<img src="https://bbs-static.miyoushe.com/static/2024/10/24/4e51d950ff7bbf9c4e9f036ecd912977_4521275005250735270.png" alt="耍酷" width="75" >

**Reply by March-777th** [reply]
能帮上忙就好<img src="https://bbs-static.miyoushe.com/static/2024/05/10/7abd8d0e930e86bda4faa08ba826a937_6450886582959431871.png" alt="三月七 1周年" width="75" >

**Comment by BiancaAtagina** [comment]
感谢三月，不然真的搞不懂。

**Comment by spoopop** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/06/19/3472710a3b57940c4e56f828fda903af_331180519040296987.png" alt="花火 锤飞" width="75" >
好用

**Reply by spoopop** [reply]
哦哦

**Comment by BSJ3456A** [comment]
手机端软件里账号登不进去发不了帖怎么办 救救QAQ

**Reply by kind-style** [reply]
？

**Reply by kind-style** [reply]
可以发啊

**Comment by kunga555** [comment]
求助求助，老师，我之前在绳网上发的帖子在绳网APP里看不见，我已经登录了，求助

**Reply by MQZCZBCK** [reply]
因为你发的那个不是帖子，而是反馈问题，发帖功能好像已经关了来着<img src="https://bbs-static.miyoushe.com/static/2025/05/08/9943254dd8ce081868395bb200388511_1046377693388090953.png" alt="爱可菲-惬意" width="75" >

**Comment by Ailian-baobao** [comment]
怎么发帖呀

**Comment by hjdcy** [comment]
![Screenshot_2025-08-22-03-31-43-03_a252b927494330cdc2c8ba3b3f952e5e.jpg](https://github.com/user-attachments/assets/281655c1-3e88-41df-9854-3f71589e385c)
这是何意？

**Comment by mcnb59** [comment]
怎么现在发不了帖子了
